Makefile.toml

```diff
@@ -7,6 +7,7 @@ BUILDSYS_ROOT_DIR = "${CARGO_MAKE_WORKING_DIRECTORY}"
 BUILDSYS_BUILD_DIR = "${BUILDSYS_ROOT_DIR}/build"
 BUILDSYS_PACKAGES_DIR = "${BUILDSYS_BUILD_DIR}/rpms"
 BUILDSYS_IMAGES_DIR = "${BUILDSYS_BUILD_DIR}/images"
+BUILDSYS_ARCHIVES_DIR = "${BUILDSYS_BUILD_DIR}/archives"
```

Comment:
Hmm, archives sounds like old files, or random tar files, or perhaps packages.  Maybe we should call them build... kits?  ;)